src/server/routes/helper.js

```diff
@@ -209,6 +357,10 @@ module.exports = {
   isOrganiser,
   isSponsor,
   exportApplicantInfo,
+  writeToCSV,
+  cherryPick,
+  bfs,
+  buildVertices,
```

Comment:
There is no need to export some of these functions, as they are never used outside of this file.